### DISCUSSION TITLE
Document [node] process.d.ts/HRTime

### DISCUSSION THREAD

**Discussion by jamesfoster-excelpoint** [DefinitelyTyped/DefinitelyTyped#69870] [Issues with a @types package]
There's no documentation for this function:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/3bf639f2d0a4b33af714b6ab83b77ed05a739b0d/types/node/process.d.ts#L163

Documentation on nodejs.org:

https://nodejs.org/api/process.html#processhrtimetime

It would also be good mark this as `@deprecated`

[a comment by an automated account is not shown]

**Comment by hkleungai** [comment]
Hi, I just opened a PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/71370 for this issue. You may want to have a look.
